CONTRIBUTING.md

```diff
@@ -64,6 +64,33 @@ Administrative notes:
 - The [OWNERS file spec] is a useful resources in making changes.
 - Maintainers and admins must be added to the appropriate lists in both [Kustomize OWNERS_ALIASES] and [SIG-CLI Teams]. If this isn't done, the individual in question will lack either PR approval rights (Kustomize list) or the appropriate Github repository permissions (community list).
 
+## Project/Product Managers
+
+Kustomize will have opportunities to join in a project/product manager role. You can
+typically start working on this role as part of a kustomize training cohort, so please
+keep an eye out for that or reach out to the leads if you are interested in this type of
+work. Expectations for this role are:
+
+- Triage 1 feature request each week, and bring it to weekly stand-up for discussion. Feature
+requests are issues labeled kind/feature, and you can find them [here](https://github.com/kubernetes-sigs/kustomize/issues?q=is%3Aissue+is%3Aopen+kind+feature+label%3Akind%2Ffeature).
```

Comment:
No need to fix. Just FYI that https://github.com/kubernetes-sigs/kustomize/labels/kind%2Ffeature also works.

Reply:
TIL. Thanks!